x/btcstkconsumer/types/expected_keepers.go

```diff
@@ -18,6 +20,17 @@ type BankKeeper interface {
 	// Methods imported from bank should be defined here
 }
 
+type ClientKeeper interface {
+	GetClientState(ctx sdk.Context, clientID string) (ibcexported.ClientState, bool)
+}
+
+type WasmKeeper interface {
+	GetContractInfo(ctx context.Context, contractAddress sdk.AccAddress) *wasmtypes.ContractInfo
+}
+
+type ChannelKeeper interface {
+}
```

Comment:
Why do we need this channel keeper if it provides no interface? Just curious.

Reply:
Good catch, removed channel keeper dependency in btcstkconsumer